pipelines/broad/dna_seq/germline/joint_genotyping/reblocking/ReblockGVCF.changelog.md

```diff
@@ -1,3 +1,10 @@
+# 1.1.1
+2021-06-17
+
+* Updated ReblockGVCF to support reblocking as the default for germline sequencing pipelines
```

Comment:
It's seemed like there were some additional changes to reblocking in this effort? i.e. new calling intervals, updated version of GATK? Can we capture any implications of those changes in the release notes as well?

Reply:
This ended up going in in the previous PR